src/transformers/models/ibert/utils/quant_utils.py

```diff
@@ -0,0 +1,264 @@
+import math
```

Comment:
If this file was taken from fairseq, we'll need correct attribution in the copyright

Reply:
These are the ones that we implemented ourselves, and not taken from fairseq. So it would not need the copyright

Reply:
Sounds great, thanks for clarifying! Could you still add a copyright, with mentions to you and HuggingFace like in other files? Thank you!

Reply:
Sure, I will add it to all the files that we included. Thanks!